### PR TITLE
Fixed a typo in a public `extractTwoSlashCompilerOptions` method

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

**Comment by orta** [comment]
Hah, oops! Thanks @Andarist

**Comment by Andarist** [comment]
@orta the CI has failed here but it doesn't seem to be related to this PR

File: packages/sandbox/src/index.ts [hunk 1]
```diff
@@ -409,8 +409,14 @@ export const createTypeScriptSandbox = (
     lzstring,
     /** Returns compiler options found in the params of the current page */
     createURLQueryWithCompilerOptions,
+    /**
+     * @deprecated Use `getTwoSlashCompilerOptions` instead.
```
Comment (Andarist) [inline packages/sandbox/src/index.ts:413]: I have initially thought that `@typescript/sandbox` is a published package so I wasn't sure if you are OK with introducing a "breaking" change here but it seems to only be a local package so I think that removing the "deprecated" variant altogether would be preferred here. Thoughts?

Comment (orta) [reply packages/sandbox/src/index.ts:413]: It's exposed to playground plugins and folks using the sandbox via https://www.typescriptlang.org/dev/sandbox/ - so this is the right call 👍🏻

**Comment by orta** [comment]
I fixed CI via another repo this morning, so a re-run should merge 👍🏻

**Pull request merged**